### ISSUE TITLE
[WCAG 2.2 Draft Feedback] 2.5.8 Pointer Target Spacing

### ISSUE THREAD

**Issue by dshoukry** [w3c/wcag#1456]
“Success Criterion 2.5.8 Pointer Target Spacing (Level AA): For each target, there is an area with a width and height of at least 44 CSS pixels that includes it, and no other targets, except when:

- Enlarge: A mechanism is available to change the CSS pixel size of each target, or its spacing, so there is an area with a width and height of at least 44 CSS pixels that includes it, and no other targets;
- Inline: The target is in a sentence or block of text;
- User agent: The size of the target is controlled by the user agent and is not modified by the author;
- Essential: A particular presentation of the target is essential to the information being conveyed.”

Most of our comments/proposals are requests to: provide research and/or references for the reasoning behind this guidance, clarify if this applies to touch screen testing on laptops, when does a fully interactive line of text meet the exemption vs. turn into a text button and then need to meet this requirement, a request to clarify what qualifies as an essential exception, consider some of our thoughts on the questions posed about internationalization and pop-over content, provide for more illustrative examples and clarify some of the examples provided, and add a section for failure examples.

Please find detailed specifics covered in our [2.5.8 Pointer Target Spacing (Level AA) Google Doc](https://docs.google.com/document/d/1qhQYPCd4T5TudlJclr4qcwZTQSSBEiAA5SZ5tqvCwt8/edit#heading=h.rv4cak38r610).

**Comment by detlevhfischer** [comment]
Proposed Working Group answer:
Due to several concerns about the proposed SC Pointer Target Spacing, The Mobile Accessibility Task Force has thought again about the best way to express the requirement and currently proposes an alternative Success Criterion:

> Target Size (AA)
The size of the target for pointer inputs is at least 26 by 26 CSS pixels except when:
> - Inline: The target is in a sentence or block of text;
> - User Agent Control: The size of the target is determined by the user agent and is not modified by the author;
> - Essential: A particular presentation of the target is essential to the information being conveyed.

**Comment by alastc** [comment]
With regards to some of the detailed comments:

> provide research and/or references for the reasoning behind this guidance

Assuming we go with the revised approach above, there is a particular study referenced. It should be noted that this type of criteria is always going to be a somewhat arbitrary line which balances user-need and feasibility (like color contrast). For example, there is research that 100px is a good minimum for people with mobility impairments on touch screen devices. However, using that size could cause issues for other groups, and would have issues with feasibility.


> clarify if this applies to touch screen testing on laptops,

It applies to web content, including when on phones, tablets, laptops with touch-screens, laptops without touch-screens, desktops, TVs etc. We cannot specify what platform a criterion is applicable to because the scope is web content.

> clarify when a fully interactive line of text meets the exemption vs. turn into a text button and then need to meet this requirement

The exception is for "The target is in a sentence or block of text", so if there is no non-link text next to it, it cannot be in a sentence of block of text. Perhaps that would be clearer if it was "The target is **within** a sentence or block of text"?

> clarify what qualifies as an essential exception

We can add more examples, are there particular scenarios that you weren't sure about?


> consider some of our thoughts on the questions posed about internationalization and pop-over content

Will do. I think the maps example fits the essential exception.


> provide more illustrative examples and clarify some of the examples provided, 

Are there any scenarios which you or the team were not sure about?


> add a section for failure examples.

Examples will go in the examples section, we'll see if someone can add that.


> If user agents already provide a mechanism to make target sizes larger, why also require a mechanism in the web content to make target sizes larger? Please provide research and/or references for the reasoning behind this guidance.

If someone has a motor impairment and struggles to hit small targets then it will also be difficult to pinch-zoom accurately. I'm not aware of research on this specifically, but I know of participants in our testing who cannot pinch-zoom at all because they cannot use two fingers at once on screens. 

There is also a general point that when using a mechanism intended to make things bigger visually is very likely to have a negative impact if used for another purpose such as increasing target sizes. We have another issue with a complaint from a low-vision user who does not want the targets to get bigger because when they zoom in they will be too big.

**Comment by alastc** [comment]
Update for the [latest text](https://raw.githack.com/w3c/wcag/scha14-pointer-target-spacing/understanding/22/pointer-target-spacing.html).

**Comment by alastc** [comment]
The above response was approved by the working group today: https://www.w3.org/2021/02/23-ag-minutes.html#item05
